src/renderer/utils/markdown.ts

```diff
@@ -130,6 +130,60 @@ const Markdown = {
 
     },
 
+    // Wikilink
+    wikilink () {
+
+      const {path: notesPath} = Config.notes;
+
+      if ( !notesPath ) return [];
+
+      var matches = [];
+      return [
+        { // Link
+          type: 'lang',
+          regex: /(?<!```(?: *)(?:[^\s`~]*)\n(?:[\s\S]*?)|`)\[{2}(.*?\|?.*?)\]{2}/g,
+          replace (match, $1, $2) {
+            matches.push($1);
+            var n = matches.length - 1;
+            return '%PLACEHOLDER' + n + '%';
+          }
+        },
+        {
+          type: 'output',
+          filter (text) {
+                for (var i=0; i< matches.length; ++i) {
+                    var content = matches[i]
+
+                    var splits = content.split("|"); 
+
+                    if (splits.length === 1) {
+                      var wikiLink = splits[0].trim();
+                      var linkText = splits[0].trim();
+                    }
+                    else {
+                      var wikiLink = splits[1].trim();
+                      var linkText = splits[0].trim();
+                    };
+
+                    wikiLink = decodeURI ( wikiLink );
+                    var basename = [wikiLink, 'md'].join('.'); // the extension should be configurable (= default extension)
+                    var filePath = path.join ( notesPath, basename ); 
+                    var link = `<a target="_blank" href="file://${filePath}" class="note" data-filepath="${filePath}"><i class="icon xsmall">note</i>${linkText}</a>`;
```

Comment:
Instead of creating the full-fledged anchor tag you should make one that gets taken as the input of the subsequent extension, basically write `<a href="@note/foo"></a>` or something. We don't do that there's will be bugs in the future where somebody forgot to update both codes.

Reply:
fixed